### ISSUE TITLE
Foot eversion limited

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11156]
**Preferred term label:**
Foot eversion limited

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased foot eversion secondary to pain, weakness, or mechanical block; reduced foot range of motion; stiff foot

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fifth metatarsal fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001429
- label: Limited foot eversion
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: 
- def: Decreased foot eversion secondary to pain, weakness, or mechanical block.
- comment: This feature is assocaited with a reduced foot range of motion and may be experienced as a stiff foot.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11156
- synonyms: Foot eversion limited(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001430
- label: Limited foot inversion
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: 
- def: Decreased foot inversion secondary to pain, weakness, or mechanical block.
- comment: This feature is assocaited with a reduced foot range of motion and may be experienced as a stiff foot.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11156
- synonyms: Foot inversion limited(NONE)
